ReactiveCocoa/AppKit/NSButton.swift

```diff
@@ -4,8 +4,24 @@ import AppKit
 
 extension Reactive where Base: NSButton {
 
-	public var pressed: Signal<Base, NoError> {
-		return trigger.map { [unowned base = self.base] in base }
+	internal var associatedAction: Atomic<(action: CocoaAction<Base>, disposable: Disposable?)?> {
+		return associatedValue { _ in Atomic(nil) }
 	}
 
+	public var pressed: CocoaAction<Base>? {
+		get {
+			return associatedAction.value?.action
+		}
+
+		nonmutating set {
+			base.target = newValue
```

Comment:
Hey @andersio, I'm not totally sure if I understand how to use the `trigger` here instead of assigning the target/action. Any advice or clues? Thanks!